### DISCUSSION TITLE
Burger Wizard PR discussion board (Because he banned me FOR NO REISIN)

### DISCUSSION THREAD

**Discussion by destrucktoid** [Bubberstation/Bubberstation#1082] [codebase discussion]
Discussion about PR: https://github.com/Bubberstation/Bubberstation/pull/1081
and PR: https://github.com/Bubberstation/Bubberstation/pull/1078

The Wizard as an antagonist is supposed to be a nukie level threat that is a single person. As such they are given equipment that is extremely powerful by design. Now Burger, in his usual fashion, has decided to take the nuclear approach with his PR regarding adding a ban hammer and removing the high frequency blade.

Don't get me wrong, i love the idea of the ban hammer and do think it should be added to wizard, it's funny and gimmicky and definitely a good addition to the wizard kit. _I don't think removing the high frequency blade is a good change however_. The **only** thing that Burger brings up that they don't like about it that isn't just whining that it's powerful (which it is meant to be), is that it gibs. Yet instead of just doing the quick, easy and logical decision of just making it unable to gib a dead body; they decide to remove it entirely from the game and add an entirely new item.

Suggested changes: Don't remove the high frequency blade, don't nerf the high frequency blade, just make it unable to gib dead bodies. Do however add the ban hammer that shit is funny.

Next up: The Lich changes.

I don't personally think nerfs to the lich is needed, the argument for it is that it's hard for people to find the phylactery, however that is part of the point. Furthermore, there is particles that point in the direction of the phylactery already and just giving away the phylactery location after 3 deaths is, in my opinion, quite dumb. If crew cannot follow the hints already given then they frankly do not deserve to kill you fully. 

The argument that it can be hidden in space is false due to the fact that the phylactery must be on the same Z level as you to work properly, the item that is the phylactery is given away by the fact it turns black and hiding the phylactery well is something that frankly should be rewarded. Making it so that the phylactery is easy to find just makes picking bind soul useless, especially since after you die the first time it becomes easier to kill you in the follow up due to you losing all your equipment.

Suggested Changes: I don't think anything should happen with bind soul, however if we must do something to it, the most i would advocate for is making it so that if the phylactery is hit with a null rod it gets destroyed, thus making it easier to destroy by the chaplain.

**Comment by projectkepler-ru** [comment]
what is a lich
